lib/MiqVm/MiqVm.rb

```diff
@@ -345,3 +346,4 @@ def extract(c)
   vm.unmount
   puts "...done"
 end
+=end
```

Comment:
Most of this, in file  dev instance test code, hasn't been used in a very long time. If we think there will be no future need for it, it should probably be removed.

Otherwise, everything looks good to me.

Reply:
Alright, I'll update the PR and just yank it.

Reply:
@roliveri Updated.